fixtures/_integration.cpp.sem.uast

```diff
@@ -208,6 +225,27 @@
          ],
       },
    ],
-   'Prop_IncludeDirectives': [],
+   'Prop_PreprocStatements': [
+      { '@type': "uast:Import",
```

Comment:
Specifically `uast:InlineImport`... That is commented out in SDK :D Will uncomment it back.

Reply:
Ping me here when its done and I'll change it.